### ISSUE TITLE
Enhancement: Leaving/transforming items on pedastal.

### ISSUE THREAD

**Issue by darkshadow543** [Axieum/PedestalCrafting#4]
I am trying to make a recipe in which an Item is left behind on the pedestal after it is crafted.  For instance, Just to Show a recipe here goes.

//item reuse code
val bowl1 = <ore:materialBowlOfPromises0>;
val emptybowl = bowl1.transformReplace(<evilcraft:bowl_of_promises:1>);
//recipe
recipes.remove(<evilcraft:promise>);
mods.pedestalcrafting.Pedestal.addRecipe(<evilcraft:promise>, 100, <evilcraft:promise_acceptor>, [emptybowl, <xreliquary:mob_ingredient:6>], [["flame", "40"], ["crit","10"]], [["largeexplode", "50"]], [["smoke", "50"]]);

This recipe was intended to craft an upgrade for Evilcraft.  What was expected was that a empty bowl of promises would be left behind, since they are expensive.  Instead both items, it only takes the reliquary item.

This was written thinking it was not a bug, but rather a lack of support for this transformation.

**Comment by Axieum** [comment]
Hmm. I'll take a look into this; though I must admit I never even thought or knew about a`transformReplace` mechanic, though it makes sense!

So you're telling me that your code above only takes the reliquary item and doesn't even touch the replaceable item?

**Comment by darkshadow543** [comment]
Yup, thats what happens, video evidence here. https://youtu.be/1NaJK4tSdn4

**Comment by Axieum** [comment]
I had a look into this earlier today. It seems that in order to implement it, I'd need to make CraftTweaker a hard dependency which is fine. However after seeing what I'd potentially have to do to get it to work, I'd rather refactor the entire mod (re-write) and add other requested features (e.g. pedestal rotation, pedestal tier locking).

I'll leave this issue open and welcome anyone to submit a pull request should they implement it. But I don't quite have the time right now, and probably not for a while.

_I'm really sorry, and I personally would like this feature too!_
